tests/integration_tests/test_hyperopt.py

```diff
@@ -48,7 +48,7 @@
         },
         "combiner.fc_layers": {
             "type": "category",
-            "values": [[{"fc_size": 512}, {"fc_size": 256}], [{"fc_size": 512}], [{"fc_size": 256}]],
+            "values": [[{"output_size": 512}, {"output_size": 256}], [{"output_size": 512}], [{"output_size": 256}]],
```

Comment:
this default is likely too big, let's decrease it for faster tests

Reply:
Done